### DISCUSSION TITLE
I can't beat Pydantic validation for request handler

### DISCUSSION THREAD

**Discussion by nrydanov** [ag2ai/faststream#1998] [Q&A]
Sorry, but currently there's not so much docs in web and I need help.

I'm trying to use Faststream with Pydantic and accept model for request handler like this:
```python
class Input(BaseModel):
    body: str

@broker.publisher("inbrief.scraper.out.json")
@broker.subscriber("inbrief.scraper.in.json")
async def scrape(
    request: Input,
):
    logger.info("Started serving scrapping request")
```

however, when I send a message with kcat (even with -J flag) I see Pydantic validation error saying input type is bytes which is not acceptable one.

What am I doing wrong?

**Comment by Lancetnik** [comment]
@nrydanov I think, you tried to send a string, but your annotation expects `{ bode: "" }` JSON

So, the following example works fine

```python
import json

from pydantic import BaseModel

from faststream import FastStream, Logger
from faststream.kafka import KafkaBroker

broker = KafkaBroker()
app = FastStream(broker)


class Input(BaseModel):
    body: str


@broker.subscriber("in")
async def scrape(request: Input, logger: Logger):
    logger.info(request)


@app.after_startup
async def _():
    await broker._producer._producer.send(
        "in",
        json.dumps({"body": "Hello World!"}).encode(),
    )
```

To consume string body just use `request: str` annotation
